src/applications/vaos/new-appointment/components/VAFacilityPage/FacilitiesRadioWidget.jsx

```diff
@@ -81,8 +97,13 @@ export default function FacilitiesRadioWidget({
           <button
             type="button"
             className="additional-info-button va-button-link vads-u-display--block"
-            onClick={() => setDisplayAll(!displayAll)}
+            aria-atomic="true"
+            aria-live="assertive"
```

Comment:
I would remove these two aria attributes entirely here, since this button text will never change. It'll be removed once it is clicked.

Reply:
Deleted